Factory/AuthenticationServiceFactory.php

```diff
@@ -2,20 +2,15 @@
 namespace Port1HybridAuth\Factory;
 
 use Port1HybridAuth\Service\AuthenticationServiceInterface;
-use Port1HybridAuth\Service\ConfigurationServiceInterface;
+use Shopware\Components\DependencyInjection\Container;
 use SimpleSAML\Module;
-use \Shopware\Components\DependencyInjection\Container;
```

Comment:
Please leave this line in with leading backslash, since this is a correct php import statement. Also, this will work on both plugin install ways: This works for composer require and Plugin Manager installation.

Reply:
Ok, I'll check this during my tests using the different installation methods of the plugin as mentioned in the discussion of your initial pull request.

Reply:
BTW: Would it be possible to provide PHPUnit tests or other tests in order to get your tests transparent to publicity?